Tools/build/generate_sbom.py

```diff
@@ -0,0 +1,143 @@
+"""Tool for generating Software Bill of Materials (SBOM) for Python's dependencies"""
+
+import re
+import hashlib
+import json
+import glob
+import pathlib
+import typing
```

Comment:
We have some `mypy.ini` files in the repo, which are checked by `.github/workflows/mypy.yml`, if you'd like to check this file via a `Tools/mypy.ini`.

Reply:
Added in https://github.com/python/cpython/pull/112303/commits/e1e665a0b6e84145e3b139c969f71de782bc803c